src/components/registration/steps.jsx

```diff
@@ -324,7 +324,7 @@ module.exports = {
         },
         getYearOptions: function () {
             return Array.apply(null, Array(100)).map(function (v, id) {
-                var year = 2016 - id;
+                var year = 2017 - id;
```

Comment:
@rschamp 👍 Good thinking! I'm assuming we'd like to be using ReactJS to get the year, or does this typical JS solution work? http://stackoverflow.com/a/6002276/